geomet/wkt.py

```diff
@@ -88,7 +101,23 @@ def loads(string):
     # NOTE: This is not the intended purpose of `tokenize`, but it works.
     tokens = (x[1] for x in tokenize.generate_tokens(sio.readline))
     tokens = _tokenize_wkt(tokens)
-    geom_type = next(tokens)
+    geom_type_or_srid = next(tokens)
+    srid = None
+    geom_type = geom_type_or_srid
+    if geom_type_or_srid == 'SRID':
+        # The geometry WKT contains an SRID header.
+        _assert_next_token(tokens, '=')
+        srid = next(tokens)
+        try:
+            # Make sure the SRID is an integer. If not, it's invalid.
+            int(srid)
```

Comment:
Wouldn't it be better to use an int for the SRID as it is an integer.
So I would have written "srid = int(next(tokens))"

Reply:
@nicoe That's a good point, thanks for pointing that out.

I've been in contact with the maintainer of https://github.com/bryanjos/geo, which is a similar library for Elixir. We both ran into the same use case and I'd like to come with a semi-standard approach.

In any case, you're right; if the SRID is an int it might as well just be an int in the output (rather than a string).